developers/blobstream-rollups.md

```diff
@@ -0,0 +1,123 @@
+---
+description: Learn how to build rollups that use Blobstream
+---
+
+## Optimistic rollups
+
+One type of rollups that can be built with Blobstream is optimistic rollups.
+An optimistic rollup is a rollup that commits optimistically to a set of blocks,
+and allows the other parties to verify that the blocks are valid,
+and if they're not, they can create fraud proofs to signal that.
+
+Through the use of Blobstream, Celestia allows optimistic rollups to use Celestia as a DA layer,
```

Comment:
[opinion]

as far as intros, I think leading with this is best, if we're aiming at rollup devs here, then I think this would be better than prefacing with what an optimistic rollup is. We can could just say this sentence, but then we don't even have to mention optimistic rollups, just optimistic rollups in general.

if the goal of this post is to explain how to create commitments to rollups, then we should lead with that.

I think this intro also assumes that the reader is aware that the commitments are done differently, however the purpose of this doc appears to be to explain that there is a difference and this is a good starting point.